core/cli/test/index.test.ts

```diff
@@ -72,6 +72,21 @@ describe('cli', () => {
     expect(config).toHaveProperty('hookTasks.build:local.conflicting')
   })
 
+  it('should indicate when there are conflicts between plugins that are cousins in the tree', async () => {
```

Comment:
**thought**: maybe we should have another test to make sure that a plugin that is present in multiple ancestors doesn't cause a conflict?

e.g., that
```
├ frontend-app
│ ╰ webpack
╰ webpack
```
is okay, to give a contrived example.

Reply:
because of https://github.com/Financial-Times/dotcom-tool-kit/blob/main/core/cli/src/plugin.ts#L115-L118 there's no way it could conflict, but a test for that wouldn't hurt